src/probabilities_estimators/permutation_ordinal/spatial_permutation.jl

```diff
@@ -84,7 +84,14 @@ struct SpatialSymbolicPermutation{D,P,V} <: ProbabilitiesEstimator
     viewer::Vector{CartesianIndex{D}}
     arraysize::Dims{D}
     valid::V
+    m::Int
 end
+
+# helper functions to get an "m" property also for the spatial version
```

Comment:
Using these functions is fine, but you should rename  `get_m` to something more informative (in #126, I use `stencil_length`).

Reply:
Also, `get_thing` is a very pythonic way to name functions, which is rather bad. All functions "get you" some number or other thing. Name the function directly the thing it gets you. In julia functions that don't get you a thing but mofify a thing in place are anyways ending in `!` so no reason for `get_` or `set_` in Julia.